examples/floating_interdomain/usecases/FloatingVl3_autoscale/README.md

```diff
@@ -0,0 +1,250 @@
+# NSM over interdomain autoscaled vL3 network
+
+## Description
+
+This example shows how to configure autoscaled vL3-network for interdomain.
+
+The diagram is presented below.
+
+![NSM kernel2kernel Diagram](./floating_vl3_autoscale.svg "NSM Kernel2Kernel Scheme")
+
+## Requires
+
+Make sure that you have completed steps from [interdomain](../../)
+
+## Run
+
+**1. Deploy**
+
+1.1. Switch context to the *floating domain*.
+
+```bash
+export KUBECONFIG=$KUBECONFIG3
+```
+
+1.2. Start **vl3 ipam** and register **vl3 network service** in the *floating domain*.
+
+
+Note: *By default we're using ipam prefix is `169.254.0.0/16` and client prefix len is `24`. We also have two vl3 nses in this example. So we are expect to have a two vl3 addresses: `169.254.0.0` and `169.254.1.0` that should be accessible by each client.*
+
+
+```bash
+kubectl apply -k https://github.com/networkservicemesh/deployments-k8s/examples/floating_interdomain/usecases/FloatingVl3_autoscale/cluster3?ref=3175354ab685daf0b2fc5ed7b0b6673b266956d8
+```
+
+1.3. Get an IP address assigned to **vl3-ipam**.
+
+```bash
+vl3_ipam_ip=$(kubectl get services vl3-ipam -n nsm-system -o go-template='{{index (index (index (index .status "loadBalancer") "ingress") 0) "ip"}}')
+```
+
+1.4. Switch context to the *cluster1*.
+
+```bash
+export KUBECONFIG=$KUBECONFIG1
+```
+
+1.5. First we need to prepare a pod-template for the supplier with a correct **vl3 ipam URL**. Let's use `kustomize` to create it:
+
+```bash
+cat > kustomization.yaml <<EOF
+---
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+
+resources:
+- https://raw.githubusercontent.com/networkservicemesh/deployments-k8s/3175354ab685daf0b2fc5ed7b0b6673b266956d8/examples/floating_interdomain/usecases/FloatingVl3_autoscale/cluster1/pod-template-base.yaml
+
+patchesStrategicMerge:
+- patch-pod-template.yaml
+EOF
+```
+
+```bash
+cat > patch-pod-template.yaml <<EOF
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  name: nse-vl3-vpp-1
+spec:
+  containers:
+    - name: nse
+      env:
+        - name: NSM_PREFIXSERVERURL
+          value: "tcp://$vl3_ipam_ip:5006"
```

Comment:
We definitely shouldn't use static ips. 
See at comment https://github.com/networkservicemesh/deployments-k8s/pull/7054#discussion_r945282488

Reply:
Done